### PR TITLE
Track2 SDK migration for Storage account related cmdlets 

### PR DESCRIPTION
**Pull request by yifanz0**
## Description

This PR includes the initial setup and the Track2 SDK migration for Storage account related cmdlets. Some fixes for this [issue ](https://github.com/Azure/azure-sdk-for-net/issues/28853) of New-AzStorageAccount and Set-AzStorageAccount are not included. 

Besides that, following cmdlets are templately remove from storage project, and will add later.
    <Compile Remove="StorageAccount\AddAzureStorageAccountNetworkRule.cs" />
    <Compile Remove="StorageAccount\GetAzureStorageAccountNetworkRuleSet.cs" />
    <Compile Remove="StorageAccount\InvokeAzureStorageAccountFailover.cs" />
    <Compile Remove="StorageAccount\InvokeAzureStorageAccountHierarchicalNamespaceUpgrade.cs" />
    <Compile Remove="StorageAccount\RemoveAzureStorageAccountNetworkRule.cs" />
    <Compile Remove="StorageAccount\RestoreAzStorageBlobRange.cs" />
    <Compile Remove="StorageAccount\UpdateAzureStorageAccountNetworkRuleSet.cs" />

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Storage/Storage.Management/Models/PSGeoReplicationStats.cs [hunk 1]
```diff
@@ -12,32 +12,32 @@
...
 
-            PSGeoReplicationStats pSGeoReplicationStats = new PSGeoReplicationStats();
-
-            pSGeoReplicationStats.Status = geoReplicationStats.Status;
-            pSGeoReplicationStats.LastSyncTime = geoReplicationStats.LastSyncTime;
-            pSGeoReplicationStats.CanFailover = geoReplicationStats.CanFailover;
-
+            PSGeoReplicationStats pSGeoReplicationStats = new PSGeoReplicationStats
+            {
+                Status = geoReplicationStats.Status != null ? geoReplicationStats.Status.ToString() : null,
```
Comment (BethanyZhou) [inline src/Storage/Storage.Management/Models/PSGeoReplicationStats.cs:32]:
```suggestion
                Status = geoReplicationStats.Status?.ToString(),
```
```suggestion
                Status = geoReplicationStats.Status != null ? geoReplicationStats.Status.ToString() : null,
```

File: src/Storage/Storage.Management/Models/PSAzureFilesIdentityBasedAuthentication.cs [hunk 2]
```diff
@@ -46,7 +46,7 @@ public PSActiveDirectoryProperties(ActiveDirectoryProperties properties)
                 this.DomainSid = properties.DomainSid;
                 this.AzureStorageSid = properties.AzureStorageSid;
                 this.SamAccountName = properties.SamAccountName;
-                this.AccountType = properties.AccountType;
+                this.AccountType = properties.AccountType != null ? properties.AccountType.ToString() : null;
```
Comment (BethanyZhou) [inline src/Storage/Storage.Management/Models/PSAzureFilesIdentityBasedAuthentication.cs:49]:
```suggestion
                this.AccountType = properties.AccountType?.ToString() ;
```

**Review by BethanyZhou — COMMENTED** [review]
Generally looks good to me

**Comment by isra-fel** [comment]
Hi, what's the status of this PR. Who needs to follow up? Can it be merged or converted to a draft?

**Comment by yifanz0** [comment]
I think we don't need this PR anymore. Will just close the PR.

**Pull request closed without merging**